contrib/github_merge/github_merge.nit

```diff
@@ -157,23 +153,24 @@ end
 for arg in args do
 	# With a arg, merge the PR
 	var number = arg.to_i
-	var pr = curl.getpr(repo, number)
-	if pr == null then
+	var pull = api.get_pull(repo, number)
+	if pull == null then
 		print "Not a PR: {number}"
 		return
 	end
-	var revs = curl.getrev(repo, pr)
+
+	var revs = api.get_pull_reviewers(repo, pull)
 
 	var mergemsg = new Template
-	mergemsg.add "Merge: {pr["title"].to_s}\n\n"
-	mergemsg.add "{pr["body"].to_s}\n\n"
-	mergemsg.add "Pull-Request: #{pr["number"].to_s}\n"
-	for r in revs do
-		mergemsg.add "Reviewed-by: {r}\n"
+	mergemsg.add "Merge: {pull.title}\n\n"
+	mergemsg.add "{pull.body or else "N/A"}\n\n"
+	mergemsg.add "Pull-Request: #{pull.number}\n"
+	for rev in revs do
+		mergemsg.add "Reviewed-by: {rev}\n"
 	end
 	mergemsg.write_to_file("mergemsg")
 
-	var sha = pr["head"].json_as_map["sha"].as(String)
+	var sha = pull.head.sha
```

Comment:
so much grace

Reply:
The previous version was burning my eyes.